aat/src/aat/resources/features/F-079/S-508.td.json

```diff
@@ -0,0 +1,68 @@
+{
+  "_guid_": "S-508",
+  "_extends_": "F-079_Test_Base_Data",
+  "title": "must return the case type for an appropriate request",
+  "specs": [
+    "an active profile in CCD",
+    "contains a valid case type id",
+    "has the 200 OK code"
+  ],
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "Content-Type": "application/json;charset=UTF-8",
+      "Transfer-Encoding": "chunked"
+    },
+    "body": {
+      "id": "BEFTA_CASETYPE_2_1",
+      "description": "Demo Case BEFTA CASETYPE 2 1",
+      "version": {
+        "number": "[[ANY]]",
+        "live_from": "2017-01-01T00:00:00.000+0000",
+        "live_until": null
+      },
+      "name": "Demo Case BEFTA CASETYPE 2 1",
+      "jurisdiction": {
+        "id": "BEFTA_JURISDICTION_2",
+        "name": "BEFTA JURISDICTION 2",
+        "description": "Content for the BEFTA Jurisdiction 2.",
+        "live_from": "2017-01-01T00:00:00.000+0000",
+        "live_until": null,
+        "case_types": []
+      },
+      "events": "[[ANY]]",
```

Comment:
Please use fw version 1.0.22. And this. [[ANY]] will need to be changed with 1.0.22.